byway/src/main/webapp/destinations.js

```diff
@@ -180,11 +180,23 @@ function addLocationToDom(results,container){
 * fills Start location Searchbox with previously input
 */
 function updateStartDestination(locationData){
```

Comment:
Update comments within code throughout PR, along with including params passed in

Reply:
done